pkg/R/RDD.R

```diff
@@ -1374,6 +1374,42 @@ setMethod("top",
             takeOrderedElem(rdd, num, FALSE)
           })
 
+#' Pipes elements to a forked externel process.
+#'
+#' The same as `pipe()' in Spark.
+#'
+#' @param rdd the RDD whose elements are piped to the forked externel process.
+#' @param command the command to fork an externel process.
+#' @param ... other arguments passed to pipeRDD.
+#' @param env character vector of name=value strings to set environment variables.
+#' @return a new RDD created by piping all elements to a forked externel process.
+#' @rdname pipeRDD
+#' @export
+#' @examples
+#'\dontrun{
+#' sc <- sparkR.init()
+#' rdd <- parallelize(sc, 1:10)
+#' collect(pipeRDD(rdd, "cat")
+#' Output: c("1", "2", ..., "10")
+#'}
+setGeneric("pipeRDD", function(rdd, command, ...) { standardGeneric("pipeRDD") })
```

Comment:
recommend to use:
setGeneric("pipeRDD", function(rdd, command, env = list()) { standardGeneric("pipeRDD") })

Use ... will allows to pass more than 3 arguments than needed.
You can refer to sortBy().